app/Filament/Resources/GameResource/RelationManagers/AchievementsRelationManager.php

```diff
@@ -219,7 +220,8 @@ public function table(Table $table): Table
                         }),
                 ])
                     ->label('Bulk set type')
-                    ->visible(fn (): bool => $user->can('updateField', [Achievement::class, null, 'type'])),
+                    ->hidden($this->getOwnerRecord()->system->id === System::Events)
+                    ->visible(fn ($record): bool => $user->can('updateField', [Achievement::class, null, 'type'])),
```

Comment:
There's some risk here. Filament's API seems unclear about whether `hidden()` or `visible()` takes precedence, and the current precedence rules could change in a Filament update.

We may want to consider combining this into a single `visible()` and removing `hidden()`:

```php
// untested pseudocode

->visible(function ($record) use ($user) {
    // $record might work instead of $this
    if ($this->getOwnerRecord()->system->id === System::Events) {
        return false;
    }

    return $user->can('updateField', [Achievement::class, null, 'type']);
});
```